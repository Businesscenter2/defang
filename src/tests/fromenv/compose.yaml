services:
  fromenv:
    image: alpine
    environment:
      - FROM_ENV
