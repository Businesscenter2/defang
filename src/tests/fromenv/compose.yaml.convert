[
  {
    "name": "fromenv",
    "image": "alpine",
    "platform": 2,
    "internal": true,
    "environment": {
      "FROM_ENV": "exposed-from-env"
    },
    "networks": 1
  }
]