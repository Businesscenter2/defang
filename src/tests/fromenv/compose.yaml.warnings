 ! service "fromenv": missing compose directive: restart; assuming 'unless-stopped' (add 'restart' to silence)
 ! service "fromenv": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors