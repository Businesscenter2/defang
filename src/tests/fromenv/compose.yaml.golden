name: fromenv
services:
  fromenv:
    environment:
      FROM_ENV: exposed-from-env
    image: alpine
    networks:
      default: null
networks:
  default:
    name: fromenv_default
